### PR TITLE
Run `npm audit fix` to update packages

### PR DESCRIPTION
**Pull request by zachmargolis**
I ran `npm audit fix` so 🤷‍♂️ this might fix some vulnerabilities?

There are 4 left for manual review, but they're dependencies of `gulp` and `get-port-cli` which we're already on the latest versions of

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
     "serve": "^10.1.2",
     "stylelint": "^9.10.1",
     "stylelint-scss": "^3.9.2",
-    "uswds": "^2.0.3",
+    "uswds": "^2.7.0",
```
Comment (zachmargolis) [inline package.json:70]: This looks like a big change, should we be worried?

Comment (nickttng) [reply package.json:70]: @zachmargolis Yeah, we should check what has been updated between these two versions - I have a Jira ticket to research this update.

**Comment by juliaelman** [comment]
@zachmargolis would you be open to us closing this PR and make it a part of the USWDS upgrade work?

**Comment by zachmargolis** [comment]
@juliaelman yup! I'll close it, no worries

**Pull request closed without merging**